layouts/about/faq/section.html

```diff
@@ -0,0 +1,159 @@
+{{ define "head-suffix" }}
```

Comment:
My concern with this solution is that we are duplicating the content.

How about we create a data file to manage the questions and answers? 

I think this is is a really neat update and we should look at porting this back to hugo-solstice-theme once we are done!